suite-native/module-home/src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -45,6 +49,23 @@ export const HomeScreen = () => {
             refreshControl={refreshControl}
             noHorizontalPadding
         >
```

Comment:
Note: This file has been changed solely for testing purposes, once the fix is validated, it will be removed via `git ammend`

Reply:
removed by https://github.com/trezor/trezor-suite/pull/18302/commits/df36195c890072467ad77c1e8420984750b05d65